MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,961 @@
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[Picker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+ComboBox
+</td>
+<td>
+DataFormComboBoxEditor
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfComboBox](https://help.syncfusion.com/maui/combobox)'| markdownify }}
+</td>
+</tr>
+
+<tr>
+<td>
+RadioGroup
+</td>
+<td>
+DataFormRadioGroupEditor
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[RadioGroup](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+AutoComplete
+</td>
+<td>
+DataFormAutoCompleteEditor
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+{{'[SfAutoComplete](https://help.syncfusion.com/maui/autocomplete)'| markdownify }}
+
+</td>
+</tr>
+</table>
+
+## Text editor
+
+In the text editor, the [Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0) is loaded.
+
+Text editor have Keyboard type change by using `Keyboard` property and you can set maximum allowed length of input by using `MaxLength` property.
+
+## Multiline Text editor
+
+In the `MultilineText` editor, the [Editor](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-7.0) is loaded.
+
+And `MultilineText` editor height will auto expand/reduce based on the line wraps in editor , which allowing text to be readable without scrolling the editor.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [DataType(DataType.MultilineText)]
+    public String Address { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## Password editor
+
+In the password editor, the [Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0) is loaded.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [Display(ShortName = "Transaction password", Prompt = "Enter password")]
+    [DataType(DataType.Password)]
+    public string Password
+    {
+        get { return this.password; }
+        set{ this.password = value; }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## Date editor
+
+In the date editor, the [DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0) will be loaded.
+
+In [DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0), the default date value is current date displayed by default. You can also adding nullable `DateTime` data type for the date picker property in data form, which allows you to set the current date and display current date in date editor. 
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [DataType(DataType.Date)]
+    [Display(Name ="Birth Date")]
+    public DateTime? BirthDate { get; set; }
```

Comment:
Change BirthDate to EventDate and Namef field

Reply:
BirthDate changed to EventDate